tests/5.Transaction.js

```diff
@@ -85,6 +85,7 @@ describe('Sign', () => {
       from: '7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q',
       to: '7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q',
       fee: 0,
+      flatFee: false,
```

Comment:
I believe flatFee is false by default. Is it necessary to add it here?

Reply:
line removed.